ops/diagram-ci.md

```diff
@@ -0,0 +1,24 @@
+```mermaid
+sequenceDiagram
+participant Feature
+participant Master
+participant Dockerhub
+
+Master->>Feature: create feature1 branch on fork
+loop developing
+    Feature->>Feature: CI: build/test
+    Feature-->>Dockerhub: Deploy to personal registry
+end
+Feature-->>Master: Pull Request feature1
+Master->Master: CI: build/test
+Master->>Dockerhub: Deploy to itisfoundation repository
```

Comment:
Above dockerhub is refered as *registry* and below as *repository*. Would be better to use one or the other. I propose  ``docker images repository`` (DIR)

Reply:
itisfoundation is the repository, dockerhub the registry

Reply:
Actually the guys of docker call it repository in dockerhub  :-) For that reason I wanted to be a bit more precise
We can add a small clarification as a footnote: e.g. ``The terms *repository* and *registry* above refer to VCS repository in github and docker images registry in dockerhub, resp.``